RecoEgamma/ElectronIdentification/plugins/cuts/GsfEleHadronicOverEMLinearCut.cc

```diff
@@ -1,3 +1,5 @@
+
+
 #include "PhysicsTools/SelectorUtils/interface/CutApplicatorBase.h"
```

Comment:
these empty line changes in this file are unnecessary.
Please revert so that this file is untouched by this PR.

Reply:
Changes implemented here:
https://github.com/lsoffi/cmssw/blob/CMSSW_9_4_0_pre3_TnP/RecoEgamma/ElectronIdentification/plugins/cuts/GsfEleHadronicOverEMLinearCut.cc